content/blog/the-importance-of-ubiquitous-language/index.md

```diff
@@ -0,0 +1,188 @@
+---
+title: The Importance of Ubiquitous Language
+description: Exploration of Ubiquitous Language - What it means, how it helps, and how it is discovered and maintained.
+date: '2022-12-15'
+slug: ubiquitous-language
+tags:
+- team
+- architecture
+- "domain driven design"
+team:
+- Greg Ball
+---
+
+Good communication is one of the hardest things to be effective at. In relationships, in business, in teams, communication has and continues to be at the top of the list when it comes to what makes them good or bad.  Core to communication in general, is the ability to speak the same language.  On a software team, this can be accomplished by speaking a language that is **_ubiquitous_**.  
+
+> <span style="font-size: 50px; line-height: 50px">Ubiquitous</span>
+> <br/>
+> &nbsp;&nbsp;[yoo-**bik**-wi-_tuhs_] *adjective*
+> 
+> &nbsp;&nbsp;&nbsp;&nbsp;present, appearing, or **found everywhere**
+
+How can a "ubiquitous" language help how you talk to your teams, your users, and the business (stakeholder and end-users)?  What do you need to do to discover it, and how should it be used to be most effective?  Before you dive into the answers to these questions, let's start with the basics.
+
+## What is ubiquitous language?
+Ubiquitous language is a set of unambiguous vocabulary shared by all members and stakeholders of a product team.  Eric Evans talks "Ubiquitous Language" in his book [Domain Driven Design: Tackling Complexity in the Heart of Software](https://www.amazon.com/gp/product/0321125215). 
```

Comment:
When I first read this, I read unambiguous as ubiquitous and thought it was circular. Maybe use a synonym such as "clear" or "precise" instead to avoid confusion.

_Eric Evans talks "Ubiquitous Language" in his book_
should probably read
_Eric Evans talks about "Ubiquitous Language" in his book_